bin/bb8/remote_paths.py

```diff
@@ -0,0 +1,37 @@
+import re
+from os.path import join
+
+
+class RemotePaths(object):
+    def __init__(self, target_name, starport):
+        self.host = "{user}@{addr}".format(**starport)
+        self.remote_bucket_path = self._remote_bucket_path(target_name,
+                                                           starport)
+
+    def data(self, include_host=False):
+        return self.get_path("data", include_host=include_host)
+
+    def meta(self, include_host=False):
+        return self.get_path("meta", include_host=include_host)
+
+    def metadata_file(self, include_host=False):
+        return join(self.meta(include_host=include_host), "metadata.json")
+
+    def rsync_path(self):
+        return self.data(include_host=True)
```

Comment:
Am I right in thinking this is the only use of `include_host=True` in the project? In which case it seems like it would reduce complexity to just have this method be something like
``` 
     path = self.data()
     return "{host}:{path}".format(host=self.host, path=path)
```

and remove `include_host` everywhere? And maybe rename `get_path` to `get_path_on_host`. Unless I've missed some other usage of this, I always prefer to avoid optional boolean arguments!

Reply:
Good points. I think I used to have more uses, but no longer do. I'll fix this tomorrow morning.

Reply:
Done!